solutions/telemetry/console/consumer/.mvn/wrapper/MavenWrapperDownloader.java

```diff
@@ -0,0 +1,117 @@
+/*
```

Comment:
Maybe naming the folder as `plain` rather than `console` may make it more clear

Reply:
Agreed